query/api/src/main/java/org/codice/ddf/admin/api/Events.java

```diff
@@ -13,6 +13,7 @@
  */
 package org.codice.ddf.admin.api;
 
+@SuppressWarnings("squid:S1214" /* Keeping these constants in this interface */)
```

Comment:
We could change this interface into a class instead of suppressing the warning I think.

Reply:
Make it a class and keep it in the `query/api` module?

Reply:
ya :+1: no one should be adding to this class so it should be fine to export it. It will probably need a private constructor as well.